docs/workflow-implementation.md

```diff
@@ -0,0 +1,153 @@
+## 2. Implementation
+
+In this chapter, we'll create and execute the example "exam" workflow which has been introduced in the [previous chapter, "Workflow Concepts"](workflow-concepts.md). We'll do so by executing mutations on the Caluma GraphQL API - to get most out of this guide, we recommend to follow along using the interactive GraphiQL console of your local Caluma installation (see the [Getting started guide](https://github.com/projectcaluma/caluma#getting-started) for installation instructions).
+
+### Step 1: Workflow Design
+
+In this first step, we'll create the "exam" workflow including its tasks and flows. Although it would feel most natural to start by creating the workflow itself, we'll start by creating the "fill out exam" task - the reason for this is that when creating a new workflow, we have to tell caluma which task(s) to start when the workflow is started.
+
+#### 1a) Create "fill out exam" task
```

Comment:
This is not the same level as "1b" and "1c", also it is somehow not rendered as title on GH

Reply:
fixed

Reply:
ah wait looks like i broke "1d"

Reply:
weird ... maybe because it's right after a code block?